tests/licensedcode/data/licenses/copyheart.yml

```diff
@@ -0,0 +1,2 @@
+license:
+    - Copyheart
```

Comment:
Add a new lines after and use a lowercase key
Also where is the copyheart.txt test file?